SConstruct

```diff
@@ -60,8 +69,12 @@ opts.Add(
     )
 )
 
-# Must be the same setting as used for cpp_bindings
-opts.Add(EnumVariable("target", "Compilation target", "debug", allowed_values=("debug", "release"), ignorecase=2))
+# Editor and template_debug are compatbile (i.e. you can use the same binary for Godot editor builds and Godot debug templates).
```

Comment:
```suggestion
# Editor and template_debug are compatible (i.e. you can use the same binary for Godot editor builds and Godot debug templates).
```

Reply:
BTW the comment could maybe point out that `editor` builds do enable `TOOLS_ENABLED`, so that's the main reason to use them if needed for extensions with editor-specific functionality.